app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
Use single-quotes inside handlebar helpers. (See https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md).

Also you're missing the `value` parameter.